src/actions/resetPassword.actions.js

```diff
@@ -0,0 +1,48 @@
+import { toast } from 'react-toastify';
+import exceptionHandler from '../utils/exceptionHandler';
+import http from '../utils/httpService';
+import { setToken } from '../utils/authService';
+import actionTypes from '../constants/resetPassword.contants';
+
+export const contentLoading = () => ({
```

Comment:
Great work here @amaechi-chuks. Although, I noticed you are using your own `contentLoading` function. You can refactor this to use the general `contentLoading` already defined by @McHardex.  Thanks